Audible-Altimeter/include/Audible-Altimeter/audio_driver.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef AUDIBLE_ALTIMETER_AUDIO_DRIVER_HPP
+#define AUDIBLE_ALTIMETER_AUDIO_DRIVER_HPP
+#include <cstddef>
+#include <cstdint>
+
+class AudioDriver {
```

Comment:
I think the reason you have to link to Audible-Altimeter in the cmake is because of this class. I have been placing the interfaces inside of the Audible-Altimeter-Embedded library instead of this one. Thinking about it more, I think it makes more sense to have the interfaces be in the general library rather than in the embedded, and we could achieve this with a target include directory!

I am going to update my code to move the interfaces.